docs/en/reference/additionally/actions.mdx

```diff
@@ -173,90 +159,82 @@ You can get the following parameters:
 
 ---
 
-## actions.getDays()
+## onCreateDateEls()
 
 `Type: Function`
 
 `Default: null`
 
-`Options: getDays(day, date, HTMLElement, HTMLButtonElement, self) => void | null`
+`Options: onCreateDateEls(day, date, mainElement, HTMLButtonElement, self) => void | null`
```

Comment:
hmm it looks like I missed the `HTMLButtonElement`, not sure what the new name should be. Same problem with line 172 just below

Reply:
actually these onX actions internal code seem to only accept 2 arguments `(self, cal)`, so all these new action docs might need to be updated with the 2 args. I'll leave that up to you to verify. :)

Reply:
Okay, don't worry, I'll figure it out, thanks for your help)